core/transcoder.go

```diff
@@ -78,7 +78,7 @@ func (nv *NvidiaTranscoder) Transcode(fname string, profiles []ffmpeg.VideoProfi
 	opts := make([]ffmpeg.TranscodeOptions, len(profiles), len(profiles))
 	for i := range profiles {
 		o := ffmpeg.TranscodeOptions{
-			Oname:   fmt.Sprintf("%s/out_%s", nv.workDir, randName()),
+			Oname:   fmt.Sprintf("%s/out_%s", nv.workDir, common.RandName()),
```

Comment:
Probably should add a ".ts" here, because otherwise I suspect the wrong muxer is used. `Oname:   fmt.Sprintf("%s/out_%s.ts", nv.workDir, common.RandName()),`

Reply:
fixed